Dockerfile

```diff
@@ -19,17 +19,11 @@ ENV PATH=/opt/rh/go-toolset-1.12/root/usr/bin:$PATH \
 
 USER root
 ADD . /che-operator
-
-RUN case $(uname -m) in \
-       x86_64) ARCH="amd64" ;; \
-       s390x) ARCH="s390x";; \
-       ppc64le) ARCH="ppc64le";; \
-esac
-
 WORKDIR /che-operator
 
-# do no break RUN lines when building with UBI base images. https://projects.engineering.redhat.com/browse/OSBS-7398 & OSBS-7399
-RUN export MOCK_API=true && go test -mod=vendor -v ./... && OOS=linux GOARCH=$ARCH CGO_ENABLED=0 go build -mod=vendor -o /tmp/che-operator/che-operator cmd/manager/main.go
+RUN export ARCH="$(uname -m)" && if [[ ${ARCH} == "x86_64" ]]; then export ARCH="amd64"; fi && \
+    export MOCK_API=true && go test -mod=vendor -v ./... && \
+    OOS=linux GOARCH=${ARCH} CGO_ENABLED=0 go build -mod=vendor -o /tmp/che-operator/che-operator cmd/manager/main.go
```

Comment:
Shouldn't it be `GOOS=linux`?

Reply:
seems to me as well ..  
`$ go env | grep GOOS

GOOS="linux"`

Reply:
well, it was `OOS` before my PR, but fair enough, fixed.